### PR TITLE
Updated portfolio setup and mobile first 

### PR DESCRIPTION
**Pull request by faisalakb**
In this PR 
- Completed the Mobile Portfolio setup. 
- Made a mobile responsive design from figma template. 👍 
- Breakpoint set at `375px` for Mobile. 👍 
- Updated Readme and License files. 👍

### CODE REVIEW

**Comment by faisalakb** [comment]
Completed my First Mobile Setup Portfolio. Please check

File: index.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="./style.css" />
+  <link rel="stylesheet" href="https://rsms.me/inter/inter.css">
+  <title>Portfolio Website</title>
+</head>
+
+<body>
+  <header class="header1">
+    <a class="placeHolder logo"  href="./index.html">Welcome</a>
```
Comment (joskalenda) [inline index.html:15]: - [ ] Please make sure you align this section, your logo name and the hamburger are not well adjusted.
- [ ] The padding or margin between the elements as I highlighted are a bit different, please fix them to.

**Comment by faisalakb** [comment]
@joskalenda Updated social media links and adjusted the margin and padding of my headings. Please check

**Review by Shisui6 — APPROVED** [review]
Hi @faisalakb , 

# Approved 🪄
Your project is complete! There is nothing else to say other than... it's time to merge it 💎
Congratulations! 🎉 

## Highlights
- Good design ✅
- Good use of semantic tags ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will really improve your application._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**